From e54e597dbf32c02e721dec89a867087ea3778a1d Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 22 Nov 2019 09:58:11 +0800
Subject: [PATCH] input: touchscreen: vtl_ts: fix clang warning

drivers/input/touchscreen/vtl_ts/vtl_ts.c:52:31: warning: unused variable 'thread_running_flag' [-Wunused-variable]
static volatile unsigned char thread_running_flag =0;
                              ^

Change-Id: Ibaafeff284140a42ccca8a21f455fa5baeb178b6
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/input/touchscreen/vtl_ts/vtl_ts.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/input/touchscreen/vtl_ts/vtl_ts.c b/drivers/input/touchscreen/vtl_ts/vtl_ts.c
index efe833a0b63c..a261c691098f 100644
--- a/drivers/input/touchscreen/vtl_ts/vtl_ts.c
+++ b/drivers/input/touchscreen/vtl_ts/vtl_ts.c
@@ -49,7 +49,6 @@
 static DECLARE_WAIT_QUEUE_HEAD(waiter);
 //static struct task_struct *ts_thread = NULL;
 static unsigned char thread_syn_flag =0;
-static volatile unsigned char thread_running_flag =0;
 
 
 // ****************************************************************************
-- 
2.35.3

